app/config/config_prod.yml

```diff
@@ -19,7 +19,7 @@ monolog:
             handler:      nested
         nested:
             type:  stream
-            path:  "%kernel.logs_dir%/%kernel.environment%.log"
+            path:  "php://stderr"
```

Comment:
@fabpot currently, `app/console server:run` will discard stderr output, even with `--verbose` - I had a brief look yesterday to see why but haven't found the problem yet. I've made a reminder to do that; would be useful IMO (including a warning or even `ini_set` call if `error_log` is set to a file for CLI, which really it shouldn't be).